app/code/AKisilenko/ModuleLesson9/view/frontend/templates/widget/homepagebanner.phtml

```diff
@@ -0,0 +1,32 @@
+
+
+<style>
+    .button-homepage {
+        background: <?php echo $block->getData('button_color') ?>;
+        padding: 30px;
+        border-radius: 5px;
+    }
+</style>
+
+    <?php if ($block->getButtonTitle()): ?>
+        <h2 class='custom-widget-title'>
+            <?php echo $block->getButtonTitle(); ?>
+        </h2>
+    <?php endif; ?>
+
+    <?php if ($block->getBannerContent()) : ?>
+        <p class='custom-widget-content'>
+            <?= $block->getBannerContent(); ?>
+        </p>
+    <?php endif; ?>
+<div class="home-banner">
+        <?php echo $block->getLayout()
+            ->createBlock('Magento\Cms\Block\Block')
+            ->setBlockId($block->getData('select_block'))
+            ->toHtml(); ?>
+    <?php if ($block->getData('button_title')): ?>
+        <button class="button-homepage" onclick="location.href='geekhub-cms'">
+            <?php echo $block->getData('button_title') ?>
```

Comment:
- Use set/get magic method 
- onclick is bad decision, use <a class="button" href ....